test/unittests/commands/test_id2iri.py

```diff
@@ -16,12 +16,14 @@
 
 class TestIdToIri(unittest.TestCase):
     tmp_dir = Path("testdata/tmp")
-    mapping: ClassVar = {
-        "test_thing_0": "http://rdfh.ch/082E/-lRvrg7tQI6aVpcTJbVrwg",
-        "test_thing_1": "http://rdfh.ch/082E/JK63OpYWTDWNYVOYFN7FdQ",
-        "test_thing_2": "http://rdfh.ch/082E/1l63Oasdfopiujlkmn78ak",
-        "test_thing_with_ark_1": "http://rdfh.ch/082E/qwasddoiu8_6flkjh67dss",
-    }
+    mapping = MappingProxyType(
+        {
+            "test_thing_0": "http://rdfh.ch/082E/-lRvrg7tQI6aVpcTJbVrwg",
+            "test_thing_1": "http://rdfh.ch/082E/JK63OpYWTDWNYVOYFN7FdQ",
+            "test_thing_2": "http://rdfh.ch/082E/1l63Oasdfopiujlkmn78ak",
+            "test_thing_with_ark_1": "http://rdfh.ch/082E/qwasddoiu8_6flkjh67dss",
+        }
+    )
```

Comment:
This is the most pythonic way to make a dictionary immutable. Read more about it on https://adamj.eu/tech/2022/01/05/how-to-make-immutable-dict-in-python/